### ISSUE TITLE
openstack_db_instance_v1 root user / password

### ISSUE THREAD

**Issue by kwkwan** [riboseinc/terraform-provider-openstack#4]
@eugenetaranov I can't find a way to retrive the `openstack_db_instance_v1`'s root user and its password. Is there a way to do so in the terraform configuration? Thanks!

CC @ronaldtse

**Comment by eugenetaranov** [comment]
@kwkwan nope, it does not, trove api do not expose root password https://developer.openstack.org/api-ref/database/#create-database-instance by design. You may want to generate your own trove image and put ssh key here.

**Comment by ronaldtse** [comment]
@eugenetaranov in this case Trove API does allow enabling a “root” user. Could we implement this too?

https://developer.openstack.org/api-ref/database/#enable-root-user

**Comment by eugenetaranov** [comment]
@ronaldtse thanks for pointing out. Yes I can make this change, gophercloud has partial support for this (it can enable root but not disable) so needs to be patched as well.

**Comment by ronaldtse** [comment]
Thank you @eugenetaranov 👍
